odc/stac/_model.py

```diff
@@ -132,6 +132,27 @@ class ParsedItem:
     geometry: Optional[Geometry] = None
     """Footprint of the dataset."""
 
+    def geoboxes(self, bands: Optional[Sequence[str]] = None) -> Tuple[GeoBox, ...]:
```

Comment:
Why did you choose `Tuple` instead of `List`, if the return type is of variable length?

Reply:
it's variable length, but it's also bounded length, and it's most commonly a single element.

1. tuples are immutable
2. tuples are hashable